src/batch-transaction.ts

```diff
@@ -119,9 +119,7 @@ class BatchTransaction extends Transaction {
     }
     const reqOpts = codec.encodeQuery(query);
     const gaxOpts = query.gaxOptions;
-    if (gaxOpts) {
```

Comment:
This seems to be unrelated to type changes.  I dunno if this would break something or not 🤷‍♂️

Reply:
I ended up just shuffling this bit around to a custom type used for making sql & read requests.
https://github.com/googleapis/nodejs-spanner/pull/490/files/f917648aa5ff45cebf41885b879cebe93f906e1d#diff-5507e931de507ce1dd409414c1c57e1eR456

I think my end goal is to remove it from the codec file entirely but I don't think it can realistically happen until we have better typings across the board